terraspace/app/stacks/peer-e2e-testing/peer-e2e-testing.md

```diff
@@ -26,23 +26,23 @@ The following resources are used by this module:
 - [aws_iam_instance_profile.ec2_profile](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_instance_profile) (resource)
 - [aws_iam_role.ec2_role_atc](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role) (resource)
 - [aws_iam_role_policy_attachment.policy_attach](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role_policy_attachment) (resource)
-- [aws_instance.autocannon_runner](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance) (resource)
-- [aws_ami.autocannon](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami) (data source)
+- [aws_instance.peer-e2e-testing_runner](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance) (resource)
```

Comment:
I think docs need to be updated. Mixing "-" and "_"

Reply:
Done